JM/CF/Scripts/1_Core/CommunityFramework/CF_String.c

```diff
@@ -108,7 +108,7 @@ class CF_String : string
 	string PadStringFront(int length, CF_String padChar)
 	{
 		string newString = value;
-		length = newString.Length() - length;
+		length = length - newString.Length();
```

Comment:
Please open seperate reviews for seperate issues. Remove the change from this PR's branch.
That change needs an example of how it behaves right now and how it does after the change.